nixos/modules/system/boot/stage-2.nix

```diff
@@ -51,14 +61,21 @@ in
         '';
       };
 
-      readOnlyNixStore = mkOption {
```

Comment:
Maybe good to use `mkRemovedOptionModule` here so that users get a nice error message?

Reply:
its there, added in line 42

Reply:
Ah, right, I looked over it!